pkgs/tools/networking/podcast/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgconfig, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "podcast";
+  version = "0.17.5";
+
+  src = fetchFromGitHub {
+    owner = "njaremko";
+    repo = pname;
+    rev = version;
+    sha256 = "1g1a6yw0zxyc6vjlbycvwnm501b5hz1xkz2lnadscvmlhnld0cc2";
+  };
+
+  cargoSha256 = "0h97j6g988xjm0b2yv2cpzdcwzs1srznpl7h7ibqps8pbblfi5rs";
+
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
Could you also try to send this upstream? The patch is quite large.

Reply:
I've submitted a pull request upstream: https://github.com/njaremko/podcast/pull/17

Reply:
The pull request was merged, so I'm removing the patch.

Reply:
Hmm. Since that change was not yet included in a release, I would need to change the package version to an untagged commit: https://github.com/njaremko/podcast/commit/ecf96f79c765e36ca86f24dbdcc4a217914479eb

Would you recommend leaving the patch for now, or waiting for the next release? It looks like the last release was [July 19th](https://github.com/njaremko/podcast/releases/tag/0.17.5).

Reply:
There weren't to many changes so this is fine. You could also fetch it as a patch.

Reply:
Does `fetchpatch` work?

Reply:
Was a bit busy with work - I'm trying rewriting this with `fetchpatch`.

Reply:
I pushed https://github.com/NixOS/nixpkgs/commit/f9088f82e65358457a9fa51cf886d5a13b680934 to pull the upstream commit https://github.com/njaremko/podcast/commit/ecf96f79c765e36ca86f24dbdcc4a217914479eb which adds `Cargo.lock`:
```nix
  cargoPatches = [
    (fetchpatch {
      url =
        "https://github.com/njaremko/podcast/commit/ecf96f79c765e36ca86f24dbdcc4a217914479eb.diff";
      sha256 = "02pk8r30i1p66lwhnb3a112b612pri8ydd6fpccx71gs4rrl8pkp";
    })
  ];